test/watch/test/stdin.test.ts

```diff
@@ -17,6 +17,15 @@ test('quit watch mode', async () => {
   await vitest.isDone
 })
 
+test('rerun current pattern tests', async () => {
+  const pattern = 'TEST_NAME_PATTERN'
+  const vitest = await startWatchMode(`-t ${pattern}`)
```

Comment:
`startWatchMode` accepts an array of strings. A single argument separated with spaces doesn't seem to work through `execa`. Let's change this to array instead.

In the `test/watch/fixtures` there is no test case that would match `TEST_NAME_PATTERN`. Let's change this to `sum` so it matches test case in `fixtures/math.test.ts`.

```suggestion
  const vitest = await startWatchMode('-t', 'sum')
```